content/concepts/kata/translations.md

```diff
@@ -37,8 +37,8 @@ Translation does not have to be finished in one go. It can be saved and kept in
 
 **NOTE:** kata authors and users with appropriate privileges are able to add new language versions directly with kata editor. This approach is strongly discouraged for following reasons:
```

Comment:
- "directly with kata editor" --> "directly within the kata editor"
- "discouraged for following reasons" --> "discouraged for the following reasons"

Reply:
It seems that I missed the opportunity to add suggestion to batch since I only edited the comment to use suggestion blocks after publishing the review. Apologies for the inconvenience caused :sweat_smile: